### ISSUE TITLE
[validate list] Software

### ISSUE THREAD

**Issue by yesn0t** [NordicGamerFE/usefulmods#32]
### Notes
- to suggest a mod not on this list please open a new issue using the [Software Addition](https://github.com/NordicGamerFE/usefulmods/issues/new?assignees=&labels=Addition+Software&template=software-addition.yml&title=%5BSoftware+Addition%5D) template.
- for collaborators : please mark as done :ballot_box_with_check: the mods that have been added to their corresponding lists by clicking on the empty checkbox :black_square_button: . If further discussion is needed, you may want to convert the task item into a new issue.

### Software/tools

- [ ] https://github.com/Mathias-Boulay/Java-Mods-Optimizer (waiting on creator to make changes to documents)
- [ ] https://www.curseforge.com/minecraft/customization/zenscript-syntax-highlighter-for-notepad
- [ ] https://www.curseforge.com/minecraft/customization/zenscript-notepad-plus-plus
- [ ] https://github.com/IoeCmcomc/MCDatapacker
- [ ] https://github.com/garet90/MotorMC
- [ ] https://github.com/Podrum/Podrum
- [ ] https://github.com/Incendo/awesome-minecraft
- [ ] https://github.com/MCreator/MCreator
- [ ] https://github.com/mcourteaux/CraftMania
- [ ] https://github.com/LoomDev/Loom
- [ ] https://github.com/fhebuterne/McScanKt
- [ ] https://github.com/wrrulos/MCPTool
- [ ] https://github.com/mrkite/minutor
- [ ] https://github.com/tryashtar/image-map
- [ ] https://bridge-core.app
- [ ] https://marketplace.visualstudio.com/items?itemName=BlockceptionLtd.blockceptionvscodeminecraftbedrockdevelopmentextension
- [ ] https://docs.microsoft.com/en-us/minecraft/creator/documents/commonlyusedtools
- [ ] https://tthread.com
- [ ] https://notepad-plus-plus.org/downloads/
- [ ] https://code.visualstudio.com/Download
- [ ] https://atom.io
- [ ] https://skmedix.pl/sklauncher (Trigger windows defender)
- [ ] https://www.electronjs.org/apps/helioslauncher (Highly interesting)
- [ ] https://www.mcpatcher.net/en/hdtexturefix
- [ ] https://essential.gg
- [ ] https://pvplounge.com
- [ ] https://cosmicclient.com
- [ ] https://batmod.com
- [ ] https://www.universalminecrafteditor.com
- [ ] https://5zigreborn.eu
